set.mm

```diff
@@ -64074,7 +64074,7 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
     CKUFUDTAUCCLMNOPABCRBAUARS $.
 
   $( A relation is a one-to-one onto function iff its converse is a one-to-one
-     onto function with domain and range interchanged.  (Contributed by NM,
+     onto function with domain and codomain interchanged.  (Contributed by NM,
```

Comment:
Here I'd say range. This has to do with the set.mm definition of a function as a set of ordered pairs (as opposed to the alternative definition in which the codomain is part of the definition of the function).

For example, if ` B C_ C ` then `C` is a codomain (in the sense that `F : A --> C`), but `F` is onto `B`, and not onto `C`. It makes sense to speak of interchanging `A` and `B` (as in this theorem), but not intrerchanging `A` and `C`.

Reply:
Although I prefer "codomain" if a functio is given in the form, `F : A -*-> B . I am fine with the slash solution "codomain/range". I'll change this and the next comment with my next commit.